buildSrc/src/main/groovy/com/github/okbuilds/core/model/JavaLibTarget.groovy

```diff
@@ -22,6 +23,14 @@ class JavaLibTarget extends JavaTarget {
         } else {
             retrolambda = null
         }
+
+        // SqlDelight
+        if (project.plugins.hasPlugin('com.squareup.sqldelight')) {
+            sqldelight = new Scope(project, ["provided"] as Set,
+                    [new File(project.projectDir, "build/generated/source/sqldelight")] as Set)
```

Comment:
build/generated/source/sqldelight does not exist at project configuration time right?

Also what about debug vs release variants for the generated sources?